server/src/main/java/org/apache/druid/server/coordinator/loading/HttpLoadQueuePeon.java

```diff
@@ -390,8 +386,10 @@ public void stop()
       stopped = true;
 
       // Cancel all queued requests
```

Comment:
nit: move this comment inside the if block since we do the cancelation conditionally now; even better remove the comment as the info log explains it already.